check_api/src/main/java/com/google/errorprone/scanner/ErrorProneScanner.java

```diff
@@ -907,6 +907,21 @@ public Void visitWildcard(WildcardTree tree, VisitorState visitorState) {
     return super.visitWildcard(tree, state);
   }
 
+  protected Throwable getError(Suppressible s, Throwable t) {
+    if (t instanceof ErrorProneError) {
+      return t;
+    }
+    if (t instanceof CompletionFailure) {
```

Comment:
```suggestion
    } else if (t instanceof CompletionFailure) {
```

or use `||`

Reply:
Oh let me add a comment here, I did it this way for a reason.